packages/devtools_app/lib/src/screens/app_size/app_size_controller.dart

```diff
@@ -139,6 +146,22 @@ class AppSizeController {
   TreemapNode? _decreasedDiffTreeRoot;
   TreemapNode? _combinedDiffTreeRoot;
 
+  Map<String, dynamic>? get _activeAppUnit {
```

Comment:
maybe rename to `_dataForAppUnit` or `_activeAppData` since it doesn't return an instance of `AppUnit`, but rather the data associated with the `AppUnit`

Reply:
Good point, switched to `_dataForAppUnit`